guides/common/modules/ref_variables-for-generation-of-conversion-data.adoc

```diff
@@ -0,0 +1,36 @@
+[id="variables-for-generation-of-conversion-data_{context}"]
+= Variables for Generation of Conversion Data
+
+Before you run the Ansible role to generate conversion data, configure values of the following required Ansible variables.
+
+{Project} imports most of the required Ansible variables from the `{ansible-namespace}.convert2rhel` role.
+However, some of the required variables are not imported.
+These variables are marked with an asterisk * in the table below.
+You must create those additional variables manually and assign them to the `{ansible-namespace}.convert2rhel` role.
+
+.Required variables for conversion
+[cols="20%,5%,20%",options="header"]
+|====
+| Name | Type | Intent and value
+| `{project-context}_server_url` * | string | URL of your {ProjectServer}
+| `{project-context}_username` * | string | Your admin login, such as `admin`
+| `{project-context}_password` * | string | Your admin password
```

Comment:
Is username/password the only authentication? That feels like a bad UX and possibly very insecure. Foreman has support for API tokens so are they supported?

Reply:
Yes, only login/pass is supported. I checked with Leos.